chain/store/name.go

```diff
@@ -89,7 +203,7 @@ func (sdb *StateDB) getName(registrant []byte) (string, error) {
 }
 
 func (sdb *StateDB) getRegistrant(name string) ([]byte, error) {
-	nameId := getNameId(name)
+	nameId := "test"
 
 	if v, ok := sdb.nameRegistrants.Load(nameId); ok {
 		if registrant, ok := v.([]byte); ok {
```

Comment:
`getRegistrant` just only sub-information of `getNameInfo`. We should reused `getNameInfo` implementation at here.

Reply:
Agreed, we should rename the new `getNameInfo` method to `getRegistrant` and remove the current `getRegistrant` method.

Reply:
will keep this method in the old file